soccer/src/soccer/strategy/agent/agent_action_client.cpp

```diff
@@ -393,10 +396,11 @@ void AgentActionClient::receive_response_callback(
         rj_convert::convert_from_ros(response.get()->agent_response);
 
     for (u_int32_t i = 0; i < buffered_responses_.size(); i++) {
+
         if (buffered_responses_[i].associated_request == agent_response.associated_request) {
-            // add the robot id in the corresponding (increasing) location in the received_robot_ids
```

Comment:
is there a reason this comment was removed? is it inaccurate somehow?

Reply:
looks like all the changes to this file were formatting. unless you needed to change this file for your work, uncommit these changes too.